x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/ApiKeyIntegTests.java

```diff
@@ -131,7 +131,6 @@ private void awaitApiKeysRemoverCompletion() throws Exception {
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/47958")
     public void testCreateApiKey() throws Exception{
         final Instant start = Instant.now();
```

Comment:
```suggestion
         final Instant start = Instant.ofEpochMilli(Instant.now().toEpochMilli());
```

Reply:
I like that, I'll keep it and remove the nano removing part below